web/src/components/auth/PasswordAuthentication.vue

```diff
@@ -64,12 +64,13 @@ export default {
     },
     login() {
       this.$api.post('web/auth/passwd', this.credentials)
-          .then(responseData => {
+          .then(() => {
             console.log("Password authentication successful")
             this.redirectAfterLogin()
           })
           .catch(response => {
-            this.loginError = response.statusText
+            console.log("Password authentication failed: " + response)
+            this.loginError = response
```

Comment:
Isn't the user now confronted with JSON as error message?

Reply:
No, because `api.js` parses the response as JSON